R/model.R

```diff
@@ -71,20 +71,20 @@ carbon_fluxes_internal <-
       if (MBdecay == "DD") {
         assert_that(dd.beta > 1)
         fluxes[["F6"]] = function(B) {
-          (1 - p.ep - p.em) * 0.4 * V.d * (B ^ dd.beta)
+          (1 - p.ep - p.em) * p.b * V.d * (B ^ dd.beta)
```

Comment:
I think using p.d as microbial biomass decay is fine. Now in this equation for F6,  there's no need to include V.d, because p.d=0.4*V.d

Reply:
ah hmm okay so does that mean that equations F7.ep and F7.em should not include V.d?

Reply:
> ah hmm okay so does that mean that equations F7.ep and F7.em should not include V.d?
No, it should not include V.d

Reply:
> Does setting p.d=0.4*V.d save us a parameter? Would it make sense to save the 0.4 as a parameter? Or is this a value we would expect never to change?

No, we would not save 0.4 as a parameter, it would change and now that is incorporated with p.d